flask_datadog.py

```diff
@@ -169,12 +172,21 @@ def after_request(self, response):
         if self.config['DATADOG_RESPONSE_AUTO_TAG']:
             self.add_request_tags(['status_code:%s' % (response.status_code, )])
 
-        # Emit our timing metric
-        self.statsd.timing(self.config['DATADOG_RESPONSE_METRIC_NAME'],
+        tags = self.get_request_tags()
+
+        # Emit timing metric
+        self.statsd.timing(self.config['DATADOG_RESPONSE_TIME_METRIC_NAME'],
                            elapsed,
-                           self.get_request_tags(),
+                           tags,
                            self.config['DATADOG_RESPONSE_SAMPLE_RATE'])
 
+        # Emit response size metric
+        size = len(response.data)
```

Comment:
Instead you could do `int(response.headers['content-length'])` since it should already have been computed.

I am not sure under what conditions `content-length` is *not* available, so maybe something like this would be better?

```python
if 'content-length' in response.headers:
    size = int(response.headers['content-length'])
    self.statsd.increment(...)
```